addons/event_sale/tests/test_event_sale.py

```diff
@@ -56,7 +63,8 @@ def setUp(self):
             'event_registration_ids': [(0, 0, {
                 'event_id': event.id,
                 'name': 'Administrator',
-                'email': 'abc@example.com'
+                'email': 'abc@example.com',
+                'sale_order_line_id': self.sale_order.order_line.id,
```

Comment:
Side note: event creation hereabove could be linked on self (self.event = ...) so that it can be easily accessed in other tests